datasets/wiki_hop/wiki_hop.py

```diff
@@ -0,0 +1,118 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""WikiHop: Reading Comprehension with Multiple Hops"""
+
+from __future__ import absolute_import, division, print_function
+
+import json
+import os
+
+import datasets
+
+
+_CITATION = """\
+@misc{welbl2018constructing,
+      title={Constructing Datasets for Multi-hop Reading Comprehension Across Documents},
+      author={Johannes Welbl and Pontus Stenetorp and Sebastian Riedel},
+      year={2018},
+      eprint={1710.06481},
+      archivePrefix={arXiv},
+      primaryClass={cs.CL}
+}
+"""
+
+_DESCRIPTION = """\
+WikiHop is open-domain and based on Wikipedia articles; the goal is to recover Wikidata information by hopping through documents. \
+The goal is to answer text understanding queries by combining multiple facts that are spread across different documents.
+"""
+
+_URL = "https://drive.google.com/uc?export=download&id=1ytVZ4AhubFDOEL7o7XrIRIyhU8g9wvKA"
+
+
+class WikiHopConifg(datasets.BuilderConfig):
+    """BuilderConfig for WikiHop."""
+
+    def __init__(self, masked=False, **kwargs):
+        """BuilderConfig for WikiHop.
+
+        Args:
+          masked: `bool`, original or maksed data.
+          **kwargs: keyword arguments forwarded to super.
+        """
+        super(WikiHopConifg, self).__init__(**kwargs)
+        self.masked = masked
+
+
+class WikiHop(datasets.GeneratorBasedBuilder):
+    """WikiHop: Reading Comprehension with Multiple Hops"""
+
+    VERSION = datasets.Version("1.0.0")
+    BUILDER_CONFIGS = [
+        WikiHopConifg(
+            name="original",
+            version=datasets.Version("1.0.0"),
+            description="The un-maksed WikiHop dataset",
+            masked=False,
+        ),
+        WikiHopConifg(
+            name="masked", version=datasets.Version("1.0.0"), description="Masked WikiHop dataset", masked=True
+        ),
+    ]
+
+    DEFAULT_CONFIG_NAME = "original"
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "query": datasets.Value("string"),
```

Comment:
could this be named `question` instead for consistency with other QA datasets ?

Reply:
`query` seems better suited here since these are not exactly questions but are small queries. But no strong feeling about that.

Reply:
what do you think ?

Reply:
Will name it `question`, wont' be an issue.